Jenkinsfile

```diff
@@ -93,10 +93,8 @@ def per_exec_ws(folder) {
 def init_git() {
   // Add more info about job node
   sh (
-    script: """
-     echo "INFO: NODE_NAME=${NODE_NAME} EXECUTOR_NUMBER=${EXECUTOR_NUMBER}"
-     """,
-     label: 'Show executor node info',
+    script: './task/scripts/task_show_node_info.sh',
```

Comment:
```suggestion
    script: './tests/scripts/task_show_node_info.sh',
```

Reply:
(Side note: makes total sense to expect this not to live under `tests`, it'd be good start moving stuff to a root level `scripts`/`tools`/`stuff` folder given the variety of scripts we're adding)